management-api-server/pom.xml

```diff
@@ -183,7 +183,7 @@
                         <CASSANDRA_HOME>${basedir}/target/apache-cassandra-${cassandra.version}</CASSANDRA_HOME>
                     </systemPropertyVariables>
                     <environmentVariables>
-                        <JVM_EXTRA_OPTS>-javaagent:${basedir}/../management-api-agent/target/datastax-mgmtapi-agent-${project.version}.jar</JVM_EXTRA_OPTS>
+                        <JVM_EXTRA_OPTS>-javaagent:${basedir}/../management-api-agent/target/datastax-mgmtapi-agent-3.x-${project.version}.jar</JVM_EXTRA_OPTS>
```

Comment:
Does this need a 4x version?

Reply:
As it stands now, it does not. This bit configures any unit tests that are run to have JVM_EXTRA_OPTS set, but none of the unit tests that run currently need this to be set to the 4.x agent. Most of the tests that are run are technically integration tests (though they run in the maven unit test phase), and spin up Docker containers with the appropriate image for the Cassandra/DSE version they are testing. The Docker images will only have the appropriate version of the agent jar in them in those tests.